test/wallet.test.ts

```diff
@@ -0,0 +1,129 @@
+import { AddressInterface } from './../src/types';
+import { UtxoInterface, Wallet, walletFromAddresses } from './../src/wallet';
+import {
+  ECPair,
+  networks,
+  payments,
+  TxOutput,
+  Transaction,
+} from 'liquidjs-lib';
+import { faucet, fetchUtxos, fetchTxHex, mint } from './_regtest';
+import * as assert from 'assert';
+import { Swap } from '../src/swap';
+import { toAssetHash, unblindOutput } from '../src/utils';
+
+const network = networks.regtest;
+
+const proposerKeypair = ECPair.fromWIF(
+  'cPNMJD4VyFnQjGbGs3kcydRzAbDCXrLAbvH6wTCqs88qg1SkZT3J',
+  network
+);
+
+const proposerBlindKeypair = ECPair.fromWIF(
+  'cRdrvnPMLV7CsEak2pGrgG4MY7S3XN1vjtcgfemCrF7KJRPeGgW6',
+  network
+);
+
+const responderKeypair = ECPair.fromWIF(
+  'cSv4PQtTpvYKHjfp9qih2RMeieBQAVADqc8JGXPvA7mkJ8yD5QC1',
+  network
+);
+
+const responderBlindKeypair = ECPair.fromWIF(
+  'cVcDj9Td96x8jcG1eudxKL6hdwziCTgvPhqBoazkDeFGSAR8pCG8',
+  network
+);
+
+const proposer = payments.p2wpkh({
+  pubkey: proposerKeypair.publicKey,
+  blindkey: proposerBlindKeypair.publicKey,
+  network,
+});
+
+const responder = payments.p2wpkh({
+  pubkey: responderKeypair.publicKey,
+  blindkey: responderBlindKeypair.publicKey,
+  network,
+});
+
+const proposerAddressInterface: AddressInterface = {
+  confidentialAddress: proposer.confidentialAddress!,
+  blindingPrivateKey: proposerBlindKeypair.privateKey!.toString('hex'),
+};
+
+jest.setTimeout(15000);
+
+describe('Wallet - Transaction builder', () => {
+  describe('SwapRequest transaction', () => {
+    let proposerUtxos: UtxoInterface[];
+    let shitcoin: string;
+
+    it('should let the Proposer to create a valid SwapRequest transaction', async () => {
+      // found the proposer account with LBTC
+      await faucet(proposer.confidentialAddress!);
+      proposerUtxos = await fetchUtxos(proposer.confidentialAddress!);
+      // mint for the responder
+      shitcoin = await mint(responder.confidentialAddress!, 100);
```

Comment:
better altcoin to be more gentle? 😂

Reply:
I like bold people :D

Reply:
is "shitcoin" vulgar? as English is not my native language I sometimes find it hard to realize if it is lol.

Reply:
no its just @altafan wich is a shitcoin holder, so he feels attacked :)